Extensions/trashbin.js

```diff
@@ -303,8 +303,19 @@
 			songs: trashSongList,
 			artists: trashArtistList
 		};
-		navigator.clipboard.writeText(JSON.stringify(data));
-		Spicetify.showNotification("Copied to clipboard");
+
+		navigator.clipboard.writeText(JSON.stringify(data))
+      .catch(() => {
+        const tempElement =document.createElement("textarea");
+        tempElement.value =JSON.stringify(data);
+        
+        document.body.append(tempElement);
+        tempElement.select();
+        document.execCommand("copy");
+        
+        tempElement.remove();
+      })
+      .finally(() => Spicetify.showNotification("Copied to clipboard"));
```

Comment:
```suggestion
		await Spicetify.Platform.ClipboardAPI.copy(JSON.stringify(data));
		Spicetify.showNotification("Copied to clipboard"));
```

Reply:
I didn't know about that Spicetify.Platform.ClipboardAPI.
you forgot to make the function async tho to use await

Reply:
> I didn't know about that Spicetify.Platform.ClipboardAPI.
> 
> you forgot to make the function async tho to use await

The API method is asynchronous, also it doesn't necessarily have to be asynchronous function to use await.

Reply:
> I didn't know about that Spicetify.Platform.ClipboardAPI. you forgot to make the function async tho to use await

It was a suggestion of an alternative method, not that I completely fixed it.